src/nft/components/explore/Table.tsx

```diff
@@ -149,7 +149,8 @@ export function Table<D extends Record<string, unknown>>({
       <table {...getTableProps()} className={styles.table}>
         <thead className={styles.thead}>
           {headerGroups.map((headerGroup) => (
-            <tr {...headerGroup.getHeaderGroupProps()} key={headerGroup.id}>
+            // eslint-disable-next-line react/jsx-key
+            <tr {...headerGroup.getHeaderGroupProps()}>
```

Comment:
the table row should be given a valid key instead of removing and disabling the error

Reply:
The getHeaderGroupProps function return a key prop